dev/io.openliberty.persistence.3.1.thirdparty/src/org/eclipse/persistence/asm/ASMFactory.java

```diff
@@ -14,6 +14,8 @@
 //     Oracle - initial API and implementation
 package org.eclipse.persistence.asm;
 
+import org.eclipse.persistence.asm.internal.Util;
```

Comment:
Copyright at top should include 2024:

Copyright (c) 2023, 2024 Oracle and/or its affiliates. All rights reserved.